### PR TITLE
LPS-170067 | Quarantine tests | Master

### PR DESCRIPTION
**Pull request by sirfabios**
https://issues.liferay.com/browse/LPS-170067
https://issues.liferay.com/browse/LPS-170068
https://issues.liferay.com/browse/LPS-170069
https://issues.liferay.com/browse/LPS-170072
https://issues.liferay.com/browse/LPS-170073
https://issues.liferay.com/browse/LPS-170074
https://issues.liferay.com/browse/LPS-170083
https://issues.liferay.com/browse/LPS-170084
https://issues.liferay.com/browse/LPS-170085
https://issues.liferay.com/browse/LPS-170086
https://issues.liferay.com/browse/LPS-170089
https://issues.liferay.com/browse/LPS-170091
https://issues.liferay.com/browse/LPS-170092

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase [hunk 1]
```diff
@@ -960,6 +960,7 @@ definition {
 	@description = "LRQA-53660 Automate LPS-87571 [Reports] Filter the list of items for a certain process"
 	@priority = "5"
 	test FilterAllItemsList {
+		property portal.upstream = "quarantine";
```
Comment (evanilson-santana) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase:963]: Hey @sirfabios I think that the property for P5 tests is `property portal.acceptance = "quarantine";`

**Comment by sirfabios** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase (hunk 1 above)
Comment (sirfabios) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase]: > Hey @sirfabios I think that the property for P5 tests is `property portal.acceptance = "quarantine";`

Done

**Comment by andreefarias** [comment]
ci:forward

**Pull request closed without merging**